frontend/src/components/RotateCredentials.vue

```diff
@@ -0,0 +1,469 @@
+        return 'Another operation has already been scheduled. Only one oepration at a time can be scheduled.'
+      }
+      return ''
+    },
+    isHibernationPreventingMaintenance () {
+      return this.isShootStatusHibernated &&
+        includes(['rotate-credentials-start',
+          'rotate-etcd-encryption-key-start',
+          'rotate-credentials-complete',
+          'rotate-etcd-encryption-key-complete',
+          'rotate-serviceaccount-key-start',
+          'rotate-serviceaccount-key-complete'],
+        this.operation)
+    },
+    icon () {
+      if (this.isScheduledForMaintenance) {
+        return 'mdi-clock-outline'
+      }
+      switch (this.mode) {
+        case 'init':
+          return 'mdi-rotate-right'
+      }
+      return 'mdi-refresh'
+    },
+    confirmButtonText () {
+      if (this.isScheduledForMaintenance && !this.maintenance) {
+        return 'Unschedule Rotation'
+      }
+      if (this.maintenance) {
+        return 'Schedule Roatation'
+      }
+      switch (this.mode) {
+        case 'init':
+          return 'Initiate Roatation'
+        case 'complete':
+          return 'Complete Rotation'
+      }
+      return 'Rotate'
+    },
+    tooltip () {
+      if (this.isScheduledForMaintenance) {
+        return 'This operation is scheduled to be performed during the next maintenance time window'
+      }
+      if (this.isHibernationPreventingMaintenance && this.isShootStatusHibernated) {
+        return 'Cluster is hibernated. Wake up cluster to perform this operation.'
+      }
+      return undefined
+    },
+    componentTexts () {
+      const componentTexts = {
+        'rotate-kubeconfig-credentials': {
+          caption: this.isLoading ? 'Scheduling kubeconfig credentials rotation' : 'Start Kubeconfig Rotation',
+          errorMessage: 'Could not start the rotation of kubeconfig credentials',
+          successMessage: `Rotation of kubeconfig credentials started for ${this.shootName}`,
+          heading: 'Do you want to start the rotation of kubeconfig credentials?',
+          alert: {
+            type: 'warning',
+            messages: [
+              'The current kubeconfig credentials will be revoked.'
+            ]
+          }
+        },
+        'rotate-ca-start': {
+          caption: this.isLoading
+            ? 'Initiating certificate authorities rotation'
+            : this.isDisabled
+              ? 'Rotation already initiated'
+              : 'Initiate Certificate Authorities Rotation',
+          errorMessage: 'Could not initiate the rotation of certificate authorities',
+          successMessage: `Rotation of certificate authorities initiated for ${this.shootName}`,
+          heading: 'Do you want to initiate the rotation of certificate authorities?',
+          alert: {
+            type: 'info',
+            messages: [
+              'New CAs will be created and added to the bundle (together with the old CAs). Client certificates will be re-issued immediately.'
```

Comment:
```suggestion
              'New CAs will be created and added to the bundle (together with the old CAs).'
```

the part with `Client certificates will be re-issued immediately.` is not true for every case: 

> Shoots created with Gardener >= v1.45 have a dedicated client CA which verifies the legitimacy of client certificates. For older Shoots, the client CA is equal to the cluster CA. With the first CA rotation, such clusters will get a dedicated client CA as well.

Not sure how endusers / or even the dashboard would be able to "know" if a shoot was created with Gardener >= v1.45

Maybe we could also add `Upon success, all parties (end-users, CD pipelines etc.) should make sure to update the kubeconfig for this Shoot cluster`?

Reply:
See comment above

Reply:
done